client/src/vars.ts

```diff
@@ -9,6 +9,7 @@ export const SPACER_OPTIONS = {
 
 export const ACCESS_TOKEN_KEY = 'auth.accessToken';
 export const ACCESS_TOKEN_EXPIRY_KEY = 'auth.accessTokenExpiry';
+export const REMEMBERED_USERNAME = 'remember.username';
```

Comment:
I think `auth.rememberUsername` is a better name but this is very much a nitpick